### DISCUSSION TITLE
After upgrading package, running into a couple of issues.

### DISCUSSION THREAD

**Discussion by richardryangarcia** [wevm/viem#130] [Question] [answered]
This may be something I'm missing on my end but thought I'd ask to see if there was anything obvious that stood out to yall. Im still having trouble getting my basic node application up and running. https://github.com/richardryangarcia/bookish-octo-palm-tree

I pushed the update which upgraded the version. all regenerated node modules and pnpm-lock. 

When running `npx tsc`
<img width="858" alt="image" src="https://user-images.githubusercontent.com/31261908/223260569-005b806d-0f49-4bc5-8ef0-1b7f65edd5e4.png">

Then when running `pnpm start`
<img width="854" alt="image" src="https://user-images.githubusercontent.com/31261908/223261499-ce8d54b7-ec0d-47d6-a49d-f82b7173309c.png">

Again, this may be a tsconfig issue or something minor but thought I'd ask anyways. Excited to try out this new library!

**Comment by greenlucid** [comment]
Same issue on my end, no clue how to make progress. I tried changing stuff around the package.json and tsconfig.json but no luck.

**Reply by jxom** [reply]
Our bad. Looks like one of our dependent packages, [`abitype`](https://github.com/wagmi-dev/abitype), does not export a CJS bundle. Fixing now.

https://github.com/wagmi-dev/abitype/pull/101

**Reply by jxom** [reply]
For the meantime, to fix the compilation issue (`npx tsc`), can you add the following to your `tsconfig.json`:

```diff
{
  "compilerOptions": {
    "module": "commonjs",
    "esModuleInterop": true,
    "target": "es6",
    "moduleResolution": "node",
+   "preserveSymlinks": true,
+   "skipLibCheck": true,
    "sourceMap": true,
    "outDir": "dist"
  },
}
```

Will provide an update when we have fixed the `abitype` issue.

**Comment by jxom** [comment, marked as answer]
@richardryangarcia – can you update to `viem@0.1.2` and try again? Also make sure to include the above attributes in your `tsconfig.json`.

**Reply by richardryangarcia** [reply]
@jxom that worked! thanks for the help in getting this resolved.

**Reply by jxom** [reply]
woohoo! enjoy
